README.md

```diff
@@ -310,6 +310,7 @@ available from the Electron API.  See the following options for usage.
     
     -w | --outputWait          Integer – Time to wait (in MS) between page load and PDF creation.  
                                          If used in conjunction with -e this will override the default timeout of 10 seconds    
+    --ignoreCertificateErrors  Booler to accept self-signed and untrusted certificates
```

Comment:
`Boolean` is misspelled,  and please update the documentation to:
```
--ignoreCertificateErrors  Boolean - If true, all certificate errors thrown by Electron will be ignored.  This can be used to accept self-signed and untrusted certificates.  You should be aware of the security implications of setting this flag.
                             false - default
```

Reply:
Done! Thank you for your patience.